audio-analysis/sendPitch.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import sys
+import os
+import lib.etl.dfHelper as dfHelper
+import lib.utils.videoHelper as videoHelper
+from lib.etl.sqlConnection import sqlConnectionSetup, urlExists, urlExistsSpecific, getEngine
+
+# Run unit tests first
+os.system('python test.py -v')
+
+# Pass `python main.py log` as optional arg to see print statements
+TESTING = False
+if len(sys.argv) == 2:
+    if sys.argv[1] == 'log':
+        TESTING = True
+
+
+engine = getEngine(TESTING)
+
+url = input("Provide YT Link: ")
```

Comment:
Can we remove input prompts. I find its faster to add the yt url in the args. `sendPitch.py [yt url] log`